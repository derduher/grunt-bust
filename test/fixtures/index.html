<!DOCTYPE html>
<!--[if IE 8]> <html class="no-js lt-ie9" lang="en"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"> <!--<![endif]-->
<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>
    <title>Ubiquiti Account</title>
    <base href="/" />
    <link rel="stylesheet" href="test/fixtures/normalize.css" />
    <link rel="stylesheet" href="test/fixtures/app.css" />
</head>
<body>

    <script src="test/fixtures/require.js"></script>
    <script>
    require(['test/fixtures/common.js'], function () {
    });
    </script>
</body>
</html>
