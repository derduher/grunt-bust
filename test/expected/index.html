<!DOCTYPE html>
<!--[if IE 8]> <html class="no-js lt-ie9" lang="en"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"> <!--<![endif]-->
<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>
    <title>Ubiquiti Account</title>
    <base href="/" />
    <link rel="stylesheet" href="test/fixtures/normalize.08dbf08504465a046a5c756ff6a74c10.css" />
    <link rel="stylesheet" href="test/fixtures/app.b183f5a229ce7305565a0cfc7376a7cc.css" />
</head>
<body>

    <script src="test/fixtures/require.b4aaf2fe4fc1cd30ea75f99504f28b90.js"></script>
    <script>
    require(['test/fixtures/common.6030bec2badaea52a7c1e25f7685a88d.js'], function () {
    });
    </script>
</body>
</html>
